python/mxnet/gluon/loss.py

```diff
@@ -408,7 +408,7 @@ class KLDivLoss(Loss):
 
     .. math::
 
-        L = \sum_i {label}_i * \big[\log({label}_i) - {pred}_i\big]
+        L = \sum_i {label}_i * \big[\log({label}_i) - \log({pred}_i)\big]
```

Comment:
It looks correct at first glance. However, I've checked the implementation again and it looks that the previous version is correct.

Reply:
We are assuming the the preds are generated by the log-softmax activation.